crp_APL/controllers/LQR_on_Rails/launch/lqr_on_rails.launch.py

```diff
@@ -0,0 +1,17 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    return LaunchDescription([
+        Node(
+            package='lqr_on_rails',
+            executable='lqr_controller.py',
+            name='lqr_controller',
+            output='screen',
+            parameters=[{
+                'dt': 0.03,
+                'wheel_base': 1.9,
+                'max_steer_tire_angle': 0.436332
```

Comment:
I am just thinking, this parameter should be defined centrally, and used in all controllers... same applies to wheel base. How could we do that? What do you think @AnonymDavid @mesmatyi ?

Reply:
yes, that would be the goal, but till we define these params globally i would keep it like that

Reply:
althoug we should make at least a task for it